packages/waku/src/lib/middleware/rsc.ts

```diff
@@ -42,7 +42,10 @@ export const rsc: Middleware = (options) => {
           contentType: headers['content-type'] || '',
         };
         const readable = await (ctx.devServer
-          ? ctx.devServer.renderRscWithWorker(args)
+          ? ctx.devServer.renderRscWithWorker({
+              ...args,
+              initialModuleGraph: ctx.devServer.initialModuleGraph,
+            })
```

Comment:
We should make `args` for both for DEV and PRD:
```suggestion
          ? ctx.devServer.renderRscWithWorker(args, {
              initialModuleGraph: ctx.devServer.initialModuleGraph,
            })
```